src/storybook/config.js

```diff
@@ -107,10 +107,10 @@ function loadStories() {
   require("./stories/products/ProductVariantPage");
 
   // Orders
-  require("./stories/orders/OrderAddressEditDialog");
   require("./stories/orders/OrderBulkCancelDialog");
   require("./stories/orders/OrderCancelDialog");
   require("./stories/orders/OrderCustomer");
+  require("./stories/orders/OrderCustomerAddressesEditDialog");
```

Comment:
It's a legacy code, if you name file `*.stories.tsx` it'll be autodiscovered by storybook and will not be needed here